models/base_models.py

```diff
@@ -0,0 +1,18 @@
+from candidate_models.base_models import base_model_pool
+from candidate_models.model_commitments import model_layers
+
+
+def get_model_list():
+    return base_model_pool.keys()
```

Comment:
would have expected a list here, i.e. `return list(base_model_pool.keys())` since the function name calls for a list

Reply:
still needs addressing